.travis.yml

```diff
@@ -41,12 +41,12 @@ matrix:
       env:
         - DEPLOY_NAME="osx-python"
 
-    - name: Linux Gcc6 - Octave template
+    - name: Linux Gcc6 - Octave NO_template
```

Comment:
Why no template here?

Reply:
There is one build with "Octave with templates" and one "Octave wihtout template".
It is just to get an overview over what is broken without looking into the logs.